pkg/api/vlabs/validate_test.go

```diff
@@ -29,103 +29,127 @@ const (
 
 func Test_OrchestratorProfile_Validate(t *testing.T) {
```

Comment:
The changes in this test are meant to make the current unit tests work with the refactoring of the validate.go file. A PR for improving and increasing unit test coverage of validation functions will follow.